src/vs/workbench/browser/parts/editor/media/multieditortabscontrol.css

```diff
@@ -193,7 +193,8 @@
 .monaco-workbench .part.editor > .content .editor-group-container > .title .tabs-container.disable-sticky-tabs > .tab.sizing-fit.sticky-shrink,
 .monaco-workbench .part.editor > .content .editor-group-container > .title .tabs-container.disable-sticky-tabs > .tab.sizing-shrink.sticky-shrink,
 .monaco-workbench .part.editor > .content .editor-group-container > .title .tabs-container.disable-sticky-tabs > .tab.sizing-fixed.sticky-shrink {
-	position: static; /** disable sticky positions for sticky compact/shrink/fixed tabs if the available space is too little */
+	position: relative; /** disable sticky positions for sticky compact/shrink/fixed tabs if the available space is too little */
+	left: 0 !important;
```

Comment:
```suggestion

	/**
	 * If sticky tabs are explicitly disabled, because width is too little, make sure
	 * to reset all styles associated with sticky tabs. This includes position, z-index
	 * and left property (which is set on the element itself, hence important is needed).
	 */
	position: relative;
	z-index: unset;
	left: unset !important;
```